crash-android-lib/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id("com.android.library")
+    kotlin("android")
+    id("zcash.android-build-conventions")
+}
+
```

Comment:
I should force test orchestrator to be used for these tests, because of a few additional test cases that I'd like to add.